.github/workflows/integration-tests.yml

```diff
@@ -51,13 +51,11 @@ jobs:
           # Monitor hash or version files in the cmake/ directory for changes.
           files: |
             cmake/*.txt
-
       - name: Decide post-submit integration test enablement
         # Only enable integration tests for post-submit when build dependency changes.
         if: github.event_name == 'push' && steps.detect-change.outputs.any_changed == 'true'
```

Comment:
Now we need to change to always run on push events?

Reply:
With help from ChatGPT, got it to work!